examples/newrelic/newrelic-daemonset.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  labels:
+    name: newrelic
+  name: newrelic
+spec:
+  template:
+    metadata:
+      labels:
+        name: newrelic
+    spec:
+      # Filter to specific nodes:
+      # nodeSelector:
+      #  app: newrelic
+      securityContext:
+        privileged: true
+      hostPID: true
+      hostIPC: true
+      hostNetwork: true
+      containers:
+        - resources:
+            limits:
```

Comment:
By specifying a request instead of limit, we give the newrelic agent a higher quality of service guarantee. If the kernel starts killing things because of out of memory, we want the last thing to go to be the monitoring.

This behavior is documented in https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/resource-qos.md